pkg/proxy/userspace/proxier.go

```diff
@@ -35,8 +35,9 @@ import (
 )
 
 type portal struct {
-	ip   net.IP
-	port int
+	ip         net.IP
+	port       int
+	isExternal bool
```

Comment:
per offline conversation with Tim, leaving it as is. This code will hopefully be deleted/deprecated once we have proxying in kernel space.